application/src/main/java/org/opentripplanner/transit/model/timetable/ScheduledTripTimes.java

```diff
@@ -300,6 +305,17 @@ public OptionalInt stopIndexOfGtfsSequence(int stopSequence) {
     return OptionalInt.empty();
   }
 
+  /**
+   * Returns a time-shifted copy of this TripTimes in which the vehicle passes the given stop index
+   * (not stop sequence number) at the given time.
```

Comment:
Discussion: I think we should be consistent when taking about stops:
 - `stop position (in pattern)` Zero based index, incremented by 1 for each stop in the stop pattern.
 - `GTFS stop sequence number` - Only pass through information in OTP
 - `stop index` - the global integer index of stops used internally in OTP and Raptor to reference stops by a single `int`. There is a 1-to-1 mapping between `stop index` and `Stop:id`
 
These terms are part of the OTP domain language and should be explained, in one place - not in random method JavaDoc. Writing what something is NOT should not be necessary and create noise.